tiles/src/test/java/com/protomaps/basemap/feature/QrankDbTest.java

```diff
@@ -26,4 +26,13 @@ void testLookup() {
     assertEquals(1234, db.get("Q1"));
     assertEquals(0, db.get("Q2"));
   }
+
+  @Test
+  void testLookupMultiple() {
+    var hashMap = new LongLongHashMap();
+    hashMap.put(1, 1234);
+    var db = new QrankDb(hashMap);
+    assertEquals(1234, db.get("Q1;Q2"));
```

Comment:
Also add test for `"1;Q2"` as well to match the example provided in the bug report?

Reply:
@nvkelso the bug report is showing the exception when this runs

`Long.parseLong(osmValue.substring(1));`

the substring(1) is stripping off the Q